api/src/test/java/org/openmrs/api/VisitServiceTest.java

```diff
@@ -408,6 +408,25 @@ public void unvoidVisit_shouldUnvoidEncountersVoidedWithVisit() {
 		assertEquals(1, encountersByVisit.size());
 	}
 	
+	/**
+	 * @see VisitService#getVisitsByPatient(Patient)
+	 */
+	@Test
+	public void getVisitsByPatient_shouldReturnEmptyVisitsForNullPatient() {
+		assertEquals(0, visitService.getVisitsByPatient( null).size());
+	}
+	
+	/**
+	 * @see VisitService#getVisitsByPatient(Patient)
+	 */
+	@Test
+	public void getVisitsByPatient_shouldReturnEmptyVisitsForNullPatientId() {
+		Patient p = new Patient(  );
+		p.setPatientId( null );
```

Comment:
remove unnecessary whitespaces. please get rid of this habit. if you look around the codebase we simply do `new Patient()` or `p.setPatiendId(null)`